install_custom_module_map/after/Pods/Local Podspecs/BananaLib.podspec.json

```diff
@@ -16,7 +16,6 @@
   },
   "license": {
     "type": "MIT",
-    "file": "LICENSE",
```

Comment:
@amorde is this expected to be deleted?

Reply:
Yes, because this spec was saying it had a license file named `LICENSE` but it actually didn't so it was failing to install a reference to it. I could have added the file instead, but did that in a different test

Reply:
ah I see

Reply:
I did add a license file to the `banana-lib` test fixture in the main project PR and used that in unit tests